phoebe/atmospheres/models.py

```diff
@@ -0,0 +1,506 @@
+import numpy as np
+import glob
+import os
+import re
+
+
+class ModelAtmosphere:
+    """
+    A parent class for handling model atmosphere data. Please note that only
+    derived classes should be instantiated.
+
+    Model atmospheres are approximations to stellar atmospheres. Each model
+    connects input parameters (for example, effective temperature, surface
+    gravity, and chemical abundances) to output parameters (for example,
+    intensities at different wavelengths and angles). The ModelAtmosphere
+    class provides a common interface for handling different model
+    atmospheres.
+
+    In order to have a model atmosphere supported, the following attributes
+    need to be defined:
+
+    * `name` (string): name of the model atmosphere
+    * `prefix` (string): prefix for the model atmosphere fits keywords
+    * `basic_axis_names` (list): names of the basic axes; basic axes are
+        axes that span the basic n-dimensional model atmosphere grid. The grid
+        can be sparsely populated, but it must be regular.
+    * `mus` (array): specific angles, mu=cos(theta), where theta is the angle
+        between the observer and the surface normal.
+    * `wls` (array): wavelengths of the model atmosphere intensities.
+    * `units` (float): intensity unit conversion factor. The intensities are
+        usually given in erg/s/cm^2/A, which is converted to W/m^3 by
+        multiplying with this factor.
```

Comment:
would we ever want to just support inputting quantities and determining this automatically?